doc/topics/packaging/testing.rst

```diff
@@ -0,0 +1,117 @@
+.. _pkging-testing:
+
+================
+Testing Packages
+================
+
+The Package Test Suite
+======================
+
+The salt repo provides a test suite for testing basic functionality of our packages at ``<repo-root>/pkg/tests/``.
+The install, upgrade, and downgrade tests can be run locally and are run on most PRs that are submitted against Salt.
+
+
+.. warning::
+
+    These tests are destructive to your system, as they install the built packages onto the
+    system. They may also install older versions in the case of upgrades or downgrades.
+    It is recommended to run these in an isolated system, preferrably a VM.
+
+Setup
+=====
+In order to run the package tests, the `relenv <https://github.com/saltstack/relative-environment-for-python>`_
+onedir and built pacakges need to be placed in the correct locations.
+
+All salt packages for the wanted testing version must be placed in ``<repo-root>/pkg/artifacts/``,
```

Comment:
**Nitpick (style, non-blocking):** Passive voice again. The fix: "Place all salt packages for the applicable testing version in..."

I also changed "wanted" to "applicable" in this rewrite because I think that might be more appropriate, but that's your call.